tests/migrations/test_optimizer.py

```diff
@@ -631,3 +632,34 @@ def test_optimize_through_fields(self):
                 migrations.CreateModel("Bar", [("width", models.IntegerField())]),
             ],
         )
+
+    def test_optimize_elidable_operation(self):
+        self.assertOptimizesTo(
+            [
+                operations.special.ElidableOperation(elidable=True),
```

Comment:
When you start with an `ElidableOperation` and you call `reduce()` with e.g. `CreateModel` `reduce()` of the `ElidableOperation` is called which correctly returns `CreateModel`. Whereas `CreateModel` tries to reduce with an elidable operation which is not possible (there is just no handling for that in `CreateModel.reduce()`).